### ISSUE TITLE
[wordpress] Installing version 16.0.5 and up gives error "The MARIADB_PASSWORD environment variable is empty or not set"

### ISSUE THREAD

**Issue by ametad** [bitnami/charts#16566]
### Name and Version

bitnami/wordpress 16.0.5 and 16.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. When I do a fresh install of the chart version 16.0.4 (and down) there is no problem at all. It works perfectly.
2. A fresh install of chart version 16.0.5 suddenly gives an error: the mariadb pod is crashing.
3. Also chart versions 16.1.0 and 16.1.1 gives the same error. 

### Are you using any custom parameters or values?

Custom `values.yaml`:

```yaml
allowOverrideNone: false
autoscaling:
  enabled: true
  maxReplicas: 10
  minReplicas: 2
  targetCPU: 80
  targetMemory: null
global:
  storageClass: cvo-nas-cm
htaccessPersistenceEnabled: true
ingress:
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt-production
    kubernetes.io/ingress.allow-http: "true"
    nginx.ingress.kubernetes.io/configuration-snippet: |
      if ($host = 'www.REDACTED' ) {
        rewrite ^ https://REDACTED$request_uri permanent;
      }
    nginx.ingress.kubernetes.io/global-rate-limit: "600"
    nginx.ingress.kubernetes.io/global-rate-limit-ignored-cidrs: REDACTED
    nginx.ingress.kubernetes.io/global-rate-limit-key: ${remote_addr}-${proxy_add_x_forwarded_for}
    nginx.ingress.kubernetes.io/global-rate-limit-window: 1m
  enabled: true
  hostname: REDACTED
  ingressClassName: nginx
  tls: true
mariadb:
  primary:
    resources:
      limits: {}
      requests:
        cpu: 50m
        memory: 256Mi
memcached:
  enabled: false
persistence:
  accessModes:
  - ReadWriteMany
  annotations:
    trident.netapp.io/snapshotPolicy: default
    trident.netapp.io/snapshotReserve: "10"
  size: 10Gi
replicaCount: 1
resources:
  limits: {}
  requests:
    cpu: 75m
    memory: 512Mi
service:
  type: ClusterIP
wordpressConfigureCache: false
wordpressPassword: REDACTED
wordpressScheme: https
```

### What is the expected behavior?

The same values used with Chart version 16.0.4 should also work for Chart version 16.0.5, when installing a new deployment (with a new name).

### What do you see instead?

The MariaDB pod is crashing all the time, and an error in the logs of the MariaDB pod:

```shell
$ k logs pod/my-test-mariadb-0
mariadb 12:00:40.10 
mariadb 12:00:40.10 Welcome to the Bitnami mariadb container
mariadb 12:00:40.11 Subscribe to project updates by watching https://github.com/bitnami/containers
mariadb 12:00:40.11 Submit issues and feature requests at https://github.com/bitnami/containers/issues
mariadb 12:00:40.12 
mariadb 12:00:40.12 INFO  ==> ** Starting MariaDB setup **
mariadb 12:00:40.15 INFO  ==> Validating settings in MYSQL_*/MARIADB_* env vars
mariadb 12:00:40.16 ERROR ==> The MARIADB_PASSWORD environment variable is empty or not set. Set the environment variable ALLOW_EMPTY_PASSWORD=yes to allow the container to be started with blank passwords. This is recommended only for development.
```

### Additional information

Perhaps a change in the underlying Helm Chart of MariaDB that needs a different value?

See: https://github.com/bitnami/charts/commit/0bf2699d2f89182395b8150b00004878d93d78dc#diff-ea55a2bd5acb3184251f072212a432c97139403ef0ef433909494fccbe85b6ce

I find it hard to understand where to look, so I ask here for some help.

**Comment by chfrank-cgn** [comment]
Same issue here for 16.1.1 and 16.0.5 on AKS v1.25.6, 16.0.4 is working fine

**Comment by chfrank-cgn** [comment]
Updating from 16.0.4 to 16.0.5 will also break the installation - mariadb will go into a CrashLoopBackOff with the same error log:

```
mariadb 10:25:27.63 Welcome to the Bitnami mariadb container
mariadb 10:25:27.63 Subscribe to project updates by watching https://github.com/bitnami/containers
mariadb 10:25:27.63 Submit issues and feature requests at https://github.com/bitnami/containers/issues
mariadb 10:25:27.63 
mariadb 10:25:27.63 INFO  ==> ** Starting MariaDB setup **
mariadb 10:25:27.65 INFO  ==> Validating settings in MYSQL_*/MARIADB_* env vars
mariadb 10:25:27.65 ERROR ==> The MARIADB_PASSWORD environment variable is empty or not set. Set the environment variable ALLOW_EMPTY_PASSWORD=yes to allow the container to be started with blank passwords. This is recommended only for development.
```

**Comment by fmulero** [comment]
Thanks a lot for reporting this issue.

The problem is related to a recent version of `mariadb` chart. The issue is currently fixed in the version [12.2.2 of that chart](https://github.com/bitnami/charts/pull/16582) In couple of hours the change will be propagated to the `wordpress` chart and a new version will be available.

Sorry for the inconvenience.

**Comment by ametad** [comment]
Just a memo: this issue is indeed fixed. A fixed version of the mariadb chart is used now.
